### PR TITLE
feat: mmkv storage

### PR DESCRIPTION
**Pull request by anhle10051996**

### CODE REVIEW

File: template/App.tsx [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
+import React from 'react'
+import {LogBox, Text} from 'react-native'
+import {SafeAreaProvider} from 'react-native-safe-area-context'
+import {Provider} from 'react-redux'
+import {PersistGate} from 'redux-persist/integration/react'
+import MainLayout from './src/MainLayout'
+import {persistor, store} from './src/store/store'
+import {injectStore} from './src/services/networking/axios'
 
 interface TextWithDefaultProps extends Text {
```
Comment (ngochuyduong) [inline template/App.tsx:10]: Should move interface to common interface folder

**Review by ngochuyduong — CHANGES_REQUESTED** [review]

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]
Should remove unrelated code from PR #7

File: template/App.tsx [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
+    underlineColorAndroid?: 'transparent'
+  }
 }
 
-(Text as unknown as TextWithDefaultProps).defaultProps =
-  (Text as unknown as TextWithDefaultProps).defaultProps || {};
-(Text as unknown as TextWithDefaultProps).defaultProps!.allowFontScaling =
-  false;
-(Text as unknown as TextWithDefaultProps).defaultProps = {
+;(Text as unknown as TextWithDefaultProps).defaultProps =
```
Comment (ThinhKimVo) [inline template/App.tsx:17]: check format again

Comment (ThinhKimVo) [reply template/App.tsx:17]: why using `unknown`?

File: template/src/services/mmkv/storage.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import {storage} from '.'
```
Comment (ThinhKimVo) [inline template/src/services/mmkv/storage.ts:1]: shouldn't `import from '.'`

File: template/src/utilities/utils.ts [hunk 4]
```diff
@@ -39,3 +39,7 @@ export function validateEmail(email: string): boolean {
 export function isDarkMode(): boolean {
   return Appearance.getColorScheme() === 'dark'
 }
+
+export function isObject<T>(val: T): boolean {
+  return typeof val === 'object' && val?.constructor !== FormData && val !== null
```
Comment (ThinhKimVo) [inline template/src/utilities/utils.ts:44]: please check error Type "Property 'constructor' does not exist on type 'NonNullable<T>'"

**Pull request closed without merging**